### DISCUSSION TITLE
Feature Request: Shift & Trace

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#236] [Ideas]
I was just thinking about a feature I'd like to see from Toon Boom Harmony and OpenToonz, and probably TVPaint as well, and that would be the ability to move (or shift) frames that are in the onion skin so you can trace something more efficiently, without having to alter the actual frames. This is an important feature to utilize when you're doing a paperless traditional animation workflow.

**Comment by keiwando** [comment]
If you haven't already, you'll probably want to add an upvote to this post, which is about the same feature: 
https://github.com/keiwando/toonsquid/discussions/49
